source/backups.rst

```diff
@@ -290,6 +400,15 @@ Following steps are needed to restore a previously saved backup:
       spec:
         pxcCluster: "cluster1"
         backupName: "backup1"
+        pitr:
+          type: date
+          date: "2020-12-31 09:37:13"
+          backupSource:
+            destination: s3://S3-BUCKET-NAME/BACKUP-NAME
+            s3:
+              credentialsSecret: my-cluster-name-backup-s3
```

Comment:
`bucket` is mandatory if I understand correctly and `destination` for `pitr/backupSource` should be removed - I tried to use it but it didn't work. Please check with @nonemax 
My understanding is that `destination` works for the full backup part of restore yaml, but not for pitr part.